src/Smalot/PdfParser/PDFObject.php

```diff
@@ -202,10 +201,11 @@ public function cleanContent(string $content, string $char = 'X')
     }
 
     /**
-     * Takes a string of PDF document stream text and formats it into
-     * a multi-line string with one PDF command on each line, separated
-     * by \r\n. If the given string is null, or binary data is detected
-     * instead of a document stream then return an empty string.
+     * @internal Takes a string of PDF document stream text and formats
+     * it into a multi-line string with one PDF command on each line,
+     * separated by \r\n. If the given string is null, or binary data
+     * is detected instead of a document stream then return an empty
+     * string.
```

Comment:
```suggestion
     * Takes a string of PDF document stream text and formats
     * it into a multi-line string with one PDF command on each line,
     * separated by \r\n. If the given string is null, or binary data
     * is detected instead of a document stream then return an empty
     * string.
     *
     * @internal
```